omod/src/main/web/dashboardwidgets/programstatus/programstatus.controller.js

```diff
@@ -390,7 +390,8 @@ export default class ProgramStatusController {
         }).then((response) => {
             // redirect to Mark Patient Dead page if specific "death" outcome is configured and selected
             if (this.markPatientDeadOutcome && this.canMarkPatientDead &&
-                response.outcome && response.outcome.uuid === this.markPatientDeadOutcome) {
+                ((response.outcome && response.outcome.uuid === this.markPatientDeadOutcome)
+                || (this.patientProgram.outcome && this.patientProgram.outcome.uuid == this.markPatientDeadOutcome))) {
```

Comment:
So I'm thinking this is saying "also redirect if the *existing* outcome was "mark patient died")?   I was thinking we should not redirect in the case where the existing outcome and the set outcome was "mark patient died"... does that handle this case?  

Thinking it through though, I guess it's also arguable that it's better to redirect in that case as well, just to be safe... and it looks like in this case it would redirect with the current death date, which indeed the user might want to be prompted to update in that case...

Reply:
Yes, it will redirect in any case where either the current status or the new status or both are "dead." I agree that if the patient status is dead and being updated e.g. to died on a different date, that they should be redirected to the death page.

Reply:
So, agreed, I like it way you have it... can you update the comment to reflect that it redirects in either of these cases?

Reply:
Yeah good catch! Done.